USB ID: add PID 0x613e and 0x613f for Geckonator bootloader

flummer opened the pull request:
I would like to request two Product IDs for my Geckonator project, which is a small development board with a SiLabs Happy Gecko microcontroller.

The Product IDs are for the USB bootloader, that will show up as a mass storage device on the host computer.

Product ID 1 is for a bootloader that programs the device itself and Product ID 2 is for a bootloader that program a connected device.

The hardware is shared under CC-BY-SA and the firmware is GPLv3

Source files are shared at:
- https://github.com/flummer/geckonator-hw/
- https://github.com/flummer/geckonator-dev/
- https://github.com/flummer/geckoboot/

Descriptive string ID 1: "Geckonator Bootloader"
Descriptive string ID 2: "Geckonator Bootloader (target)"

cuvoodoo commented:
quite good work. it just needs a couple of small changes before I can grant the USB PID:
- could you export the schematic (e.g. as PDF) for geckonator-hw and geckonator-dev? open source software only requires the source code to be present (i.e. not the binary) because this is readable using any text editor. but for open source hardware it's harder to expect everyone to have the right software to read the source files (even if in your case you used the free software KiCAD). You already exported the board gerber. The schematic is the main missing part, but any other output/fabrication files (BOM, manufacturer instructions, ...) are also welcome.
- mention the license name in the hardware files (e.g. most of the time in the schematic title box). in geckonator-hw the CC version is missing. in the geckonator-dev schematic no license is mentioned (the LICENSE file is already present though).
- it seems the geckonator-dev sources are not complete. some symbol libraries are missing (SeeedOPL, efm32). could you include these in you project files?

flummer commented:
- I have now updated the two repositories with PDF schematics, and also updated both with more detailed licensing info in the schematics (both the KiCad file and the PDFs).
- efm32.lib has been added to the geckonator-dev repository, but not the SeeedOPL. I should probably make the SeeedOPL version a separate branch and make the master have the generic KiCad components instead, but that is a slightly more elaborate project ;-)

cuvoodoo commented:
- thanks for the updated and exported schematics
- I could not find under which license the Seeed Studio Open Parts Library is and if it allows to redistribute it (e.g. put a copy in the project), but you could still refer it (and the link to it) in the README meanwhile

flummer commented:
- geckonator-dev README is now updated with links to the Seeed OPL library for KiCad
- I'll make an issue with their repo and check the license and see what the best way of linking/including the library is.
